### DISCUSSION TITLE
How about re-adding Microsoft.Solver.Foundation or better alternative?

### DISCUSSION THREAD

**Discussion by HimanshuBansal112** [dotnet/csharplang#7125] [Language Ideas]
Solving equations with constraints and other problems of equations can be important in many applications. For example, I was making a graph maker for some purpose in c# but I could not find a library for it. Though, there was a library Microsoft.Solver.Foundation which cannot be used in newer versions since outdated. I tried searching several alternative but could not found one that solves equations with constraints in simpler, faster and bug-free method. An in-built method would be a good and reasonable thing. For numbering uses of it, I also was trying to make it for graph such that it will show multiple solutions upto certain limit we want if there are multiple solutions considering every equation and for objective value as well. It would be helpful if there will be a library.

**Comment by HaloFour** [comment]
I'd suggest that sounds like a request to the [runtime repo](https://github.com/dotnet/runtime), which maintains the .NET class library.

**Reply by HimanshuBansal112** [reply]
Ok, can you please tell how to transfer this since it says there are no eligible repository to transfer to?

**Comment by HimanshuBansal112** [comment]
Moved to https://github.com/dotnet/runtime/discussions/84896
